docs/user/supported_devices.rst

```diff
@@ -400,6 +400,10 @@ Footnotes
   The device or target is reaching its end of life soon. This means that support
   in the next major release of Gluon is doubtful.
 
+.. [#featureset-tiny]
+  These devices do not support the full featureset of Gluon. Namely, WPA3 support
+  is missing on these devices.
```

Comment:
Maybe we should mention that this is only relevant for the private-wifi feature at the moment.